pkg/probe/firewall_ippool.go

```diff
@@ -0,0 +1,91 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type IpPool struct {
+	Name      string  `json:"name"`
+	IPTotal   int     `json:"natip_total"`
+	IPInUse   int     `json:"natip_in_use"`
+	Clients   int     `json:"clients"`
+	Available float64 `json:"available"`
+	Used      int     `json:"used"`
+	Total     int     `json:"total"`
+}
+
+type IpPoolResponse struct {
+	Results map[string]IpPool `json:"results"`
+	VDOM    string            `json:"vdom"`
+	Version string            `json:"version"`
+}
+
+func probeFirewallIpPool(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		mAvailable = prometheus.NewDesc(
+			"fortigate_ippool_available",
+			"Percentage available in ippool",
```

Comment:
`fortigate_ippool_available_percentage`

Reply:
used `_ratio` to match with other metrics in the project

Reply:
That's fine, and I like that - but a ratio is defined as a value between 0.0 and 1.0, whereas percentage is 0 - 100. So you need to divide the value by 100.0 in this case (which I think is a good idea).

Reply:
@bluecmd sorry it took me a while to finish this, but now this is 0 - 1.0